guides/common/modules/proc_configuring-remote-execution-for-pull-client-project-server.adoc

```diff
@@ -0,0 +1,40 @@
+[id="configuring-remote-execution-for-pull-client-project-server_{context}"]
+= Configuring Remote Execution for Pull Client on the {ProjectServer}
+
+By default, Remote Execution uses SSH as the transport mechanism for the Script provider.
+However, Remote Execution also offers pull-based transport, which you can use if your infrastructure prohibits outgoing connections from {Project} to hosts.
+
+This comprises `pull-mqtt` mode on {Project} in combination with a pull client running on hosts.
+Configure the `pull-mqtt` mode to migrate from Katello Agent, which is a deprecated method of pull-based transport.
```

Comment:
Maybe add "If you still use Katello Agent, configure ...".

Reply:
Reworded :+1: